css/navigation.css

```diff
@@ -48,13 +57,13 @@
   .nav-area ul {
     position: fixed;
     width: 100%;
-    height: 100vh;
-    background: rgb(39, 39, 39);
+    height: auto;
```

Comment:
Please make this height 100vh in order to fill up the whole screen. It ca't be halfway the screen.